docs/articles/deployment/scenarios.md

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
rename the file to `index.md` so it shows up as the default document

Reply:
done